lib/Windowing/InViewListItem.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { node, number, shape } from 'prop-types';
+
+function InViewListItem({ children, renderIndexes, index, baseStyle }) {
+  const top = (renderIndexes.start + index) * 50;
```

Comment:
Should this be?
```suggestion
  const top = (renderIndexes.start + index) * itemHeight;
```

This is the only RFC comment in this review.